### PR TITLE
Node.replaceChild must be overriden (close #264)

### LINKED ISSUE #264: Node.replaceChild must be overriden
Code for reproduce:
```javascript
var style = document.createElement('style');
var styleTextNode1 = document.createTextNode('div.class1 { background-image: url("/image1.png"); }');
var styleTextNode2 = document.createTextNode('div.class2 { background-image: url("/image2.png"); }');

style.appendChild(styleTextNode1);
style.replaceChild(styleTextNode2, styleTextNode1);
```
style.innerHtml equal 'div.class2 { background-image: url("/image2.png"); }'.
It is wrong because it contains non proxed url `/image2.png`.

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin @VasilyStrelyaev

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

File: test/client/fixtures/sandbox/node/dom-processor-test.js [hunk 1]
```diff
@@ -420,3 +420,15 @@ test('Script and style content added via a child text node must be overridden (G
     script.insertBefore(scriptTextNode2, scriptTextNode1);
     ok(script.childNodes[0].data.indexOf('var host2=__get$(__get$Loc(location),"host")') > -1);
 });
+
+test('Node.replaceChild must be overriden (GH-264)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: *overridden*
```
test('Node.replaceChild must be overridden (GH-264)', function () {
```

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
@VasilyStrelyaev check and merge

**Pull request merged**